lib/shortcuts/scheduled-lambda.js

```diff
@@ -34,15 +38,17 @@ class ScheduledLambda extends Lambda {
 
     const {
       ScheduleExpression,
+      ScheduleRoleArn,
+      ScheduleGroupName,
```

Comment:
I only exposed these, but we could expose more.

Reply:
Like `FlexibleTimeWindow` etc? I think just these is good enough.

Reply:
ya or like timezone, although eventbridge is default utc so users shouldnt notice a difference